CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+By participating to this project, you agree to abide our [code of
+conduct](/CODE_OF_CONDUCT.md).
+
+## Setup your machine
+TODO
+
+## Create a commit
+
+Commit messages should be well formatted, and to make that "standardized", we
+are using Conventional Commits.
+
+You can follow the documentation on
+[their website](https://www.conventionalcommits.org).
+
+## Submit a pull request
+
+Push your branch to your `goreleaser` fork and open a pull request against the
```

Comment:
Should point to us rather than `goreleaser` right?

Reply:
LOL :D